lib/pbench/agent/results.py

```diff
@@ -129,28 +129,7 @@ def _unlink_tarball(self, tarball: Path, msg: str) -> str:
             )
         return msg
 
-    def make_result_tb(self, single_threaded: bool = False) -> TarballRecord:
-        """Make the result tar ball from result directory.
-
-        The metadata.log file in the result directory is double checked to be
-        sure it is valid, and then the "run.raw_size" and the
-        "pbench.tar-ball-creation-timestamp" fields are added.
-
-        The tar ball is created, verified, and the MD5 sum value is generated.
-
-        Returns a named tuple consisting of the Path object of the created tar
-        ball, its length, and its MD5 checksum value.
-
-        Raises
-          - FileNotFoundError  if the result directory does not have a
-                               metadata.log file
-          - BadMDLogFormat     if the metadata.log file has a pbench.name field
-                               value which does not match the result directory
-                               name.
-          - RuntimeError       if any problems are encountered while creating
-                               or verifying the created tar ball
-        """
-        pbench_run_name = self.result_dir.name
+    def verify_metadata(self, pbench_run_name):
```

Comment:
I missed this on my previous reviews, but it would be appropriate for this function signature to have type hints and a docstring.